src/commands/DraupnirCommandDispatcher.ts

```diff
@@ -38,7 +38,7 @@ export function makeDraupnirCommandNormaliser(
   config: IConfig
 ): CommandNormaliser {
   return makeCommandNormaliser(clientUserID, {
-    symbolPrefixes: ["!"],
+    symbolPrefixes: config.commands.symbolPrefixes,
```

Comment:
Can you also check the safe mode and appservice bot mode that they all share this normaliser?

Reply:
> You cant configure any of the AS settings currently.

This is not true meow! you can, it loads this config file too!!